public/app/plugins/datasource/elasticsearch/types.ts

```diff
@@ -8,6 +8,11 @@ import {
   MetricAggregation,
   MetricAggregationType,
 } from './components/QueryEditor/MetricAggregationsEditor/aggregations';
+import {
+  PipelineMetricAggregationType,
+  BasePipelineMetricAggregation as SchemaBasePipelineMetricAggregation,
+  PipelineMetricAggregationWithMultipleBucketPaths as SchemaPipelineMetricAggregationWithMultipleBucketPaths,
+} from './dataquery.gen';
```

Comment:
Looking at the CI errors, you probably also need to do smth like `export { Elasticsearch as ElasticsearchQuery} from './dataquery.gen'`.